draft-ietf-add-dnr.xml

```diff
@@ -957,19 +941,19 @@
 
     <section title="Contributing Authors">
       <t><figure>
-          <artwork><![CDATA[   Nicolai Leymann
-   Deutsche Telekom
-   Germany
+          <artwork><![CDATA[Nicolai Leymann
```

Comment:
Again, this shouldn't be a figure.  It is text that can be presented using text elements.  I don't quite know how to get a contact format presented here; I usually just list names rather than provide all the details.